specification/immunisation-fhir-api.yaml

```diff
@@ -501,24 +521,48 @@ components:
         example: 60E0B220-8136-4CA5-AE46-1D97EF59D068
     PatientIdentifier:
       in: query
-      name: -nhsNumber
+      name: patient.identifier
       description: |
         The patient's NHS number.
-        Expressed as `<value>` where`<value>` must be a [valid NHS number](https://www.datadictionary.nhs.uk/attributes/nhs_number.html).
+        Expressed as `<type>|<value>` where `<type>` must be `https://fhir.nhs.uk/Id/nhs-number` and `<value>` must be a [valid NHS number](https://www.datadictionary.nhs.uk/attributes/nhs_number.html).
       required: true
       schema:
         type: string
-        example: "9000000009"
+        example: "https://fhir.nhs.uk/Id/nhs-number|9000000009"
     ImmunizationTarget:
       in: query
-      name: -diseaseType
+      name: -immunization.target
       description: |
         The vaccine preventable disease the dose is being administered against.
         Expressed as `<value>` where`<value>` must be a valid Snomed Code.
       required: true
       schema:
         type: string
         example: "39114911000001105"
```

Comment:
Does this example need to be COVID19, FLU, HPV or MMR?

Reply:
Yes, and I've replaced the description with a modified one from Imms History.